<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BetInfoFrame</class>
 <widget class="QFrame" name="BetInfoFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="2">
    <widget class="QLabel" name="profit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="multiplier">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="targetLabel">
     <property name="text">
      <string>Target</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="wagerLabel">
     <property name="text">
      <string>Amount Wagered</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="wager">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="multiplierLabel">
     <property name="text">
      <string>Multiplier</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="betTimeLabel">
     <property name="text">
      <string>Time of Bet</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="profitLabel">
     <property name="text">
      <string>Bet Profit</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="betTime">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="usernameLabel">
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="betInfoLabel">
     <property name="text">
      <string>Bet #</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLabel" name="clientSeedNonced">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="username">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="serverSeedHash">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="serverSeedHashLabel">
     <property name="text">
      <string>Server Seed Hash</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="rollLabel">
     <property name="text">
      <string>Roll</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="clientSeedNoncedLabel">
     <property name="text">
      <string>Nonced Client Seed</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="roll">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="target">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
